From a9f89b79c955fee65b8011aa5d8ac1b1a3e4c49b Mon Sep 17 00:00:00 2001
From: Georg Koppen <gk@torproject.org>
Date: Thu, 26 Apr 2018 13:18:27 +0000
Subject: [PATCH] Avoid cross-compilation breakage due to unwinding
 incompatibilities


diff --git a/src/bootstrap/bin/rustc.rs b/src/bootstrap/bin/rustc.rs
index 6701f58ba8..c7b03d0f92 100644
--- a/src/bootstrap/bin/rustc.rs
+++ b/src/bootstrap/bin/rustc.rs
@@ -152,7 +152,8 @@ fn main() {
         // workaround undefined references to `rust_eh_unwind_resume` generated
         // otherwise, see issue https://github.com/rust-lang/rust/issues/43095.
         if crate_name == "panic_abort" ||
-           crate_name == "compiler_builtins" && stage != "0" {
+           crate_name == "compiler_builtins" && stage != "0" ||
+           target == "i686-pc-windows-gnu" {
             cmd.arg("-C").arg("panic=abort");
         }
 
diff --git a/src/libtest/lib.rs b/src/libtest/lib.rs
index b8be1aeff1..c92ddb4e45 100644
--- a/src/libtest/lib.rs
+++ b/src/libtest/lib.rs
@@ -43,7 +43,6 @@
 extern crate getopts;
 #[cfg(any(unix, target_os = "cloudabi"))]
 extern crate libc;
-extern crate panic_unwind;
 extern crate term;
 
 pub use self::TestFn::*;
-- 
2.17.1

