From 49e52d2f748f5409917a7320dc8a8aee76385f78 Mon Sep 17 00:00:00 2001
From: Bart Simpson <bart@simpsons.net>
Date: Tue, 14 Apr 2020 13:47:50 +0200
Subject: [PATCH] mavenlocal patch


diff --git a/build.gradle b/build.gradle
index 045a19b3..2afe5b69 100644
--- a/build.gradle
+++ b/build.gradle
@@ -2,6 +2,7 @@
 
 buildscript {
     repositories {
+        mavenLocal()
         google()
         jcenter()
     }
@@ -23,6 +24,7 @@ plugins {
 
 allprojects {
     repositories {
+        mavenLocal()
         google()
         maven {
             url "https://nightly.maven.mozilla.org/maven2"
-- 
2.20.1

