From a5cfec06fb844ce6f29163f8941a42d15739c7d9 Mon Sep 17 00:00:00 2001
From: Georg Koppen <gk@torproject.org>
Date: Sun, 19 Apr 2020 17:59:16 +0000
Subject: [PATCH] git patch


diff --git a/components/support/base/build.gradle b/components/support/base/build.gradle
index b3db13a50..fe14bff5d 100644
--- a/components/support/base/build.gradle
+++ b/components/support/base/build.gradle
@@ -36,7 +36,7 @@ android {
             minifyEnabled false
             proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
 
-            buildConfigField("String", "GIT_HASH", "\"" + getGitHash() + "\"")
+            buildConfigField("String", "GIT_HASH", "\"release build\"")
         }
     }
 
-- 
2.26.1

