From 8a5a2b7d01cdc96be9857ef5895e1ac9009d6475 Mon Sep 17 00:00:00 2001
From: Georg Koppen <gk@torproject.org>
Date: Mon, 20 Apr 2020 08:03:44 +0000
Subject: [PATCH] gradle


diff --git a/build.gradle b/build.gradle
index 937057283..d85cab2b7 100644
--- a/build.gradle
+++ b/build.gradle
@@ -6,6 +6,7 @@ import static org.gradle.api.tasks.testing.TestResult.ResultType
 
 buildscript {
     repositories {
+        mavenLocal()
         google()
         jcenter()
     }
@@ -24,6 +25,7 @@ plugins {
 
 allprojects {
     repositories {
+        mavenLocal()
         google()
         jcenter()
 
diff --git a/settings.gradle b/settings.gradle
index 191de1a67..29a883ab6 100644
--- a/settings.gradle
+++ b/settings.gradle
@@ -6,6 +6,7 @@ import org.yaml.snakeyaml.Yaml
 
 buildscript {
     repositories {
+        mavenLocal()
         jcenter()
     }
 
-- 
2.26.1

