From dfda87ac67e8f61de1b524e7daf532ec113852a4 Mon Sep 17 00:00:00 2001
From: Georg Koppen <gk@torproject.org>
Date: Mon, 20 Apr 2020 08:03:44 +0000
Subject: [PATCH 5/5] gradle


diff --git a/build.gradle b/build.gradle
index a538b66ad..f4b058509 100644
--- a/build.gradle
+++ b/build.gradle
@@ -6,6 +6,7 @@ import static org.gradle.api.tasks.testing.TestResult.ResultType
 
 buildscript {
     repositories {
+        mavenLocal()
         if (project.hasProperty("googleRepo")) {
             maven {
                 name "Google"
@@ -39,6 +40,7 @@ plugins {
 
 allprojects {
     repositories {
+        mavenLocal()
         if (project.hasProperty("googleRepo")) {
             maven {
                 name "Google"
diff --git a/settings.gradle b/settings.gradle
index 54d2414aa..4b797cb13 100644
--- a/settings.gradle
+++ b/settings.gradle
@@ -12,6 +12,7 @@ pluginManagement {
 
 buildscript {
     repositories {
+        mavenLocal()
         jcenter()
     }
 
-- 
2.27.0.rc0

