From 0c37b703104a76913d90f8e7cc8eec538114dc03 Mon Sep 17 00:00:00 2001
From: Bart Simpson <bart@simpsons.net>
Date: Tue, 14 Apr 2020 14:14:28 +0200
Subject: [PATCH 4/5] kts


diff --git a/buildSrc/build.gradle.kts b/buildSrc/build.gradle.kts
index 69c23c387..b53eed13f 100644
--- a/buildSrc/build.gradle.kts
+++ b/buildSrc/build.gradle.kts
@@ -7,6 +7,7 @@ plugins {
 }
 
 repositories {
+    mavenLocal()
     jcenter()
 }
 
-- 
2.27.0.rc0

