From 8885238ddb56adddeaaa552110439e8ed1ea4e9c Mon Sep 17 00:00:00 2001
From: Georg Koppen <gk@torproject.org>
Date: Mon, 20 Apr 2020 08:26:03 +0000
Subject: [PATCH 2/5] pm app


diff --git a/settings.gradle b/settings.gradle
index 05da65f2f..54d2414aa 100644
--- a/settings.gradle
+++ b/settings.gradle
@@ -4,6 +4,12 @@
 
 import org.yaml.snakeyaml.Yaml
 
+pluginManagement {
+  repositories {
+    mavenLocal()
+  }
+}
+
 buildscript {
     repositories {
         jcenter()
-- 
2.27.0.rc0

