From 47587df8cb644bf0f7391558adf545939ee12a42 Mon Sep 17 00:00:00 2001
From: Bart Simpson <bart@simpsons.net>
Date: Tue, 14 Apr 2020 18:18:44 +0200
Subject: [PATCH 3/5] pm


diff --git a/buildSrc/settings.gradle.kts b/buildSrc/settings.gradle.kts
new file mode 100644
index 000000000..a7066edcb
--- /dev/null
+++ b/buildSrc/settings.gradle.kts
@@ -0,0 +1,5 @@
+pluginManagement {
+  repositories {
+    mavenLocal()
+  }
+}
-- 
2.27.0.rc0

