From 30ed25aeff1439d8ea981318c3d26b66984412c9 Mon Sep 17 00:00:00 2001
From: Georg Koppen <gk@torproject.org>
Date: Mon, 18 May 2020 08:59:54 +0000
Subject: [PATCH] Bug 33558: Update TOPL to use updated Android toolchain


diff --git a/android/build.gradle b/android/build.gradle
index a8d9bdc..b8442f0 100644
--- a/android/build.gradle
+++ b/android/build.gradle
@@ -40,7 +40,9 @@ publishing {
             groupId 'com.msopentech.thali.toronionproxy.android'
             artifactId 'android'
             version '0.0.3'
-            artifact(sourceJar)
+            afterEvaluate {
+              artifact(sourceJar)
+            }
             artifact("$buildDir/outputs/aar/android-release.aar")
             pom.withXml {
                 def dependenciesNode = asNode().appendNode('dependencies')
-- 
2.26.2

