From 0c30acdef6e4da186409a70300a685801b11c02d Mon Sep 17 00:00:00 2001
From: Matthew Finkel <Matthew.Finkel@gmail.com>
Date: Thu, 27 Jun 2019 20:57:21 +0000
Subject: [PATCH] libvpx provides already cpu_features


diff --git a/media/libaom/moz.build b/media/libaom/moz.build
index 2dec3de4581c..6ddc13d83c52 100644
--- a/media/libaom/moz.build
+++ b/media/libaom/moz.build
@@ -84,10 +84,10 @@ if CONFIG['OS_TARGET'] == 'Android':
     # the OS they're on, so do it for them.
     DEFINES['__linux__'] = True
 
-    if not CONFIG['MOZ_WEBRTC']:
-        SOURCES += [
-            '%%%s/sources/android/cpufeatures/cpu-features.c' % CONFIG['ANDROID_NDK'],
-        ]
+    #if not CONFIG['MOZ_WEBRTC']:
+    #    SOURCES += [
+    #        '%%%s/sources/android/cpufeatures/cpu-features.c' % CONFIG['ANDROID_NDK'],
+    #    ]
 
 for f in SOURCES:
     if f.endswith('sse2.c'):
-- 
2.26.2

