From 8091681b68c7e4439a1b111b243c38a1db506a93 Mon Sep 17 00:00:00 2001
From: Georg Koppen <gk@torproject.org>
Date: Thu, 10 Jan 2019 10:53:12 +0000
Subject: [PATCH] Bug 28875


diff --git a/dom/ipc/ContentPrefs.cpp b/dom/ipc/ContentPrefs.cpp
index ac1ea109fc9f..87d4e2da1580 100644
--- a/dom/ipc/ContentPrefs.cpp
+++ b/dom/ipc/ContentPrefs.cpp
@@ -302,6 +302,7 @@ const char* mozilla::dom::ContentPrefs::gEarlyPrefs[] = {
   "privacy.resistFingerprinting.target_video_res",
   "privacy.resistFingerprinting.video_dropped_ratio",
   "privacy.resistFingerprinting.video_frames_per_sec",
+  "privacy.spoof_english",
   "privacy.trackingprotection.lower_network_priority",
   "privacy.window.maxInnerHeight",
   "privacy.window.maxInnerWidth",
-- 
2.20.1

