From 67a9a95dc8577c4b014b14f9a6cbe803a8a6a94a Mon Sep 17 00:00:00 2001
From: Georg Koppen <gk@torproject.org>
Date: Sun, 3 Nov 2019 21:25:55 +0000
Subject: [PATCH] chop1


diff --git a/servo/components/style/Cargo.toml b/servo/components/style/Cargo.toml
index 54493548b67a..d21759b32688 100644
--- a/servo/components/style/Cargo.toml
+++ b/servo/components/style/Cargo.toml
@@ -18,11 +18,6 @@ doctest = false
 [features]
 gecko = ["nsstring", "style_traits/gecko", "fallible/known_system_malloc"]
 use_bindgen = ["bindgen", "regex", "toml"]
-servo = ["serde", "style_traits/servo", "servo_atoms", "servo_config", "html5ever",
-         "cssparser/serde", "encoding_rs", "malloc_size_of/servo", "arrayvec/use_union",
-         "servo_url", "string_cache", "crossbeam-channel", "to_shmem/servo", "servo_arc/servo"]
-gecko_debug = []
-gecko_profiler = []
 
 [dependencies]
 app_units = "0.7"
@@ -31,15 +26,12 @@ atomic_refcell = "0.1"
 bitflags = "1.0"
 byteorder = "1.0"
 cssparser = "0.25"
-crossbeam-channel = { version = "0.3", optional = true }
 derive_more = "0.13"
 new_debug_unreachable = "1.0"
-encoding_rs = {version = "0.8", optional = true}
 euclid = "0.19"
 fallible = { path = "../fallible" }
 fxhash = "0.2"
 hashglobe = { path = "../hashglobe" }
-html5ever = {version = "0.22", optional = true}
 indexmap = "1.0"
 itertools = "0.7.6"
 itoa = "0.4"
@@ -59,16 +51,11 @@ parking_lot = "0.6"
 precomputed-hash = "0.1.1"
 rayon = "1"
 selectors = { path = "../selectors" }
-serde = {version = "1.0", optional = true, features = ["derive"]}
 servo_arc = { path = "../servo_arc" }
-servo_atoms = {path = "../atoms", optional = true}
-servo_config = {path = "../config", optional = true}
 smallbitvec = "2.3.0"
 smallvec = "0.6.6"
-string_cache = { version = "0.7", optional = true }
 style_derive = {path = "../style_derive"}
 style_traits = {path = "../style_traits"}
-servo_url = {path = "../url", optional = true}
 thin-slice = "0.1.0"
 to_shmem = {path = "../to_shmem"}
 to_shmem_derive = {path = "../to_shmem_derive"}
diff --git a/toolkit/library/gtest/rust/Cargo.toml b/toolkit/library/gtest/rust/Cargo.toml
index b2e9fcb8ba3a..d3be49cdc6fa 100644
--- a/toolkit/library/gtest/rust/Cargo.toml
+++ b/toolkit/library/gtest/rust/Cargo.toml
@@ -5,27 +5,6 @@ authors = ["nobody@mozilla.org"]
 license = "MPL-2.0"
 description = "Testing code for libgkrust"
 
-[features]
-bindgen = ["gkrust-shared/bindgen"]
-servo = ["gkrust-shared/servo"]
-quantum_render = ["gkrust-shared/quantum_render"]
-cubeb-remoting = ["gkrust-shared/cubeb-remoting"]
-cubeb_pulse_rust = ["gkrust-shared/cubeb_pulse_rust"]
-gecko_debug = ["gkrust-shared/gecko_debug"]
-simd-accel = ["gkrust-shared/simd-accel"]
-moz_memory = ["gkrust-shared/moz_memory"]
-moz_places = ["gkrust-shared/moz_places"]
-spidermonkey_rust = ["gkrust-shared/spidermonkey_rust"]
-cranelift_x86 = ["gkrust-shared/cranelift_x86"]
-cranelift_arm32 = ["gkrust-shared/cranelift_arm32"]
-cranelift_arm64 = ["gkrust-shared/cranelift_arm64"]
-cranelift_none = ["gkrust-shared/cranelift_none"]
-gecko_profiler = ["gkrust-shared/gecko_profiler"]
-gecko_profiler_parse_elf = ["gkrust-shared/gecko_profiler_parse_elf"]
-bitsdownload = ["gkrust-shared/bitsdownload"]
-new_xulstore = ["gkrust-shared/new_xulstore"]
-new_cert_storage = ["gkrust-shared/new_cert_storage"]
-
 [dependencies]
 bench-collections-gtest = { path = "../../../../xpcom/rust/gtest/bench-collections" }
 mp4parse-gtest = { path = "../../../../dom/media/gtest" }
diff --git a/toolkit/library/rust/shared/Cargo.toml b/toolkit/library/rust/shared/Cargo.toml
index e6a28275c4fa..2db0c5e833bc 100644
--- a/toolkit/library/rust/shared/Cargo.toml
+++ b/toolkit/library/rust/shared/Cargo.toml
@@ -42,28 +42,6 @@ bookmark_sync = { path = "../../../components/places/bookmark_sync", optional =
 [build-dependencies]
 rustc_version = "0.2"
 
-[features]
-default = []
-bindgen = ["geckoservo/bindgen"]
-servo = ["geckoservo"]
-quantum_render = ["webrender_bindings"]
-webrender_debugger = ["webrender_bindings/webrender_debugger"]
-cubeb-remoting = ["cubeb-sys", "audioipc-client", "audioipc-server"]
-cubeb_pulse_rust = ["cubeb-sys", "cubeb-pulse"]
-gecko_debug = ["geckoservo/gecko_debug", "nsstring/gecko_debug"]
-simd-accel = ["encoding_c/simd-accel", "encoding_glue/simd-accel"]
-moz_memory = ["mp4parse_capi/mp4parse_fallible"]
-moz_places = ["bookmark_sync"]
-spidermonkey_rust = ["jsrust_shared"]
-cranelift_x86 = ["jsrust_shared/cranelift_x86"]
-cranelift_arm32 = ["jsrust_shared/cranelift_arm32"]
-cranelift_arm64 = ["jsrust_shared/cranelift_arm64"]
-cranelift_none = ["jsrust_shared/cranelift_none"]
-gecko_profiler = ["profiler_helper", "geckoservo/gecko_profiler"]
-gecko_profiler_parse_elf = ["profiler_helper/parse_elf"]
-new_xulstore = ["xulstore"]
-new_cert_storage = ["cert_storage"]
-
 [lib]
 path = "lib.rs"
 test = false
-- 
2.24.0.rc2

