From bd4a1c0f3d3e365090cf797f4d4df1860508b2f0 Mon Sep 17 00:00:00 2001
From: Georg Koppen <gk@torproject.org>
Date: Wed, 22 Apr 2020 06:09:21 +0000
Subject: [PATCH] no beam


diff --git a/mobile/android/torbrowser.configure b/mobile/android/torbrowser.configure
index ac30fde888a6..f06522e3e5e1 100644
--- a/mobile/android/torbrowser.configure
+++ b/mobile/android/torbrowser.configure
@@ -6,10 +6,6 @@
 
 # Set Tor Browser default config
 
-# Disable Android Beam (NFC integration)
-imply_option('MOZ_ANDROID_BEAM', False)
-
-
 imply_option('MOZ_ANDROID_DOWNLOAD_CONTENT_SERVICE', False)
 imply_option('MOZ_ANDROID_DOWNLOADS_INTEGRATION', False)
 imply_option('MOZ_ANDROID_EXCLUDE_FONTS', False)
-- 
2.11.0

