From 2c1f760f48d6c8b9de6a152c9dab865e5152ab3e Mon Sep 17 00:00:00 2001
From: Georg Koppen <gk@torproject.org>
Date: Tue, 29 Oct 2019 09:33:07 +0000
Subject: [PATCH 1/2] no_lto_1301


diff --git a/config/makefiles/rust.mk b/config/makefiles/rust.mk
index f2d3dddf6c03..23b44188662b 100644
--- a/config/makefiles/rust.mk
+++ b/config/makefiles/rust.mk
@@ -47,7 +47,7 @@ cargo_rustc_flags = $(CARGO_RUSTCFLAGS)
 ifndef DEVELOPER_OPTIONS
 ifndef MOZ_DEBUG_RUST
 # Enable link-time optimization for release builds.
-cargo_rustc_flags += -C lto
+# cargo_rustc_flags += -C lto
 endif
 endif
 
-- 
2.24.0.rc1

