From 66ea55701c1a3f41d643541110a9da51620a3a14 Mon Sep 17 00:00:00 2001
From: Georg Koppen <gk@torproject.org>
Date: Tue, 29 Oct 2019 10:37:53 +0000
Subject: [PATCH] min_rust 1300


diff --git a/build/moz.configure/rust.configure b/build/moz.configure/rust.configure
index 5ee7fd59084b..b0eda95ccb9f 100644
--- a/build/moz.configure/rust.configure
+++ b/build/moz.configure/rust.configure
@@ -66,7 +66,7 @@ def rust_compiler(rustc_info, cargo_info):
         You can install rust by running './mach bootstrap'
         or by directly running the installer from https://rustup.rs/
         '''))
-    rustc_min_version = Version('1.31.0')
+    rustc_min_version = Version('1.30.0')
     cargo_min_version = rustc_min_version
 
     version = rustc_info.version
-- 
2.24.0.rc1

